packages/common/src/codegen/render-solidity/common.ts

```diff
@@ -127,21 +120,33 @@ export function renderAbsoluteImports(imports: AbsoluteImportDatum[]): string {
 
 export function renderWithStore(
   storeArgument: boolean,
-  callback: (
-    _typedStore: string | undefined,
-    _store: string,
-    _commentSuffix: string,
-    _untypedStore: string | undefined,
-    _methodPrefix: string,
-    _internal?: boolean
-  ) => string
+  callback: (data: {
+    _typedStore: string | undefined;
+    _store: string;
+    _commentSuffix: string;
+    _methodNamePrefix: string;
+    _internal?: boolean;
+  }) => string
 ): string {
   let result = "";
-  result += callback(undefined, "StoreSwitch", "", undefined, "");
-  result += callback(undefined, "StoreCore", "", undefined, "_", true);
+  result += callback({ _typedStore: undefined, _store: "StoreSwitch", _commentSuffix: "", _methodNamePrefix: "" });
+  result += callback({
+    _typedStore: undefined,
+    _store: "StoreCore",
+    _commentSuffix: "",
+    _methodNamePrefix: "_",
+    _internal: true,
+  });
```

Comment:
i like the refactor to named arguments! should we omit `_typedStore` here if it's undefined? (similar to how we don't include `_untypedStore`)

Reply:
whoops, merged too hastily, will do this in a follow up

Reply:
oh and `_untypedStore` was removed because it wasn't actually used anywhere